website/docs/athena.md

```diff
@@ -4,16 +4,16 @@ title: "Amazon Athena"
 ---
 
 # Querying from Amazon Athena
-To read a OneTable synced target table (regardless of the table format) in Amazon Athena,
+To read a XTable synced target table (regardless of the table format) in Amazon Athena,
```

Comment:
are these all search/replace ? I trust that we have hit all the Onetable entries?

Reply:
Yes I have done a full project search across all website assets, documentation, and readmes. I did not touch any renames in the codebase and I think those should be taken up in a separate PR